freqtrade/plugins/pairlist/RemotePairList.py

```diff
@@ -250,6 +258,20 @@ def filter_pairlist(self, pairlist: List[str], tickers: Dict) -> List[str]:
         :return: new whitelist
         """
         rpl_pairlist = self.gen_pairlist(tickers)
-        merged_list = pairlist + rpl_pairlist
-        merged_list = sorted(set(merged_list), key=merged_list.index)
+        merged_list = []
+        filtered = []
+
+        if self._mode == "whitelist":
+            merged_list = pairlist + rpl_pairlist
```

Comment:
i know this didn't really change - but is this intentional? 

Assuming i use "volumepairlist 50" - and then "remotePairlist 5 (currently with a local file containing 3 pairs)" - would i then expect 55 pairs, or rather 5 pairs (volumepairlist further filtered onto pairs actually in the remote pairlist)?

All other pairlists would have this act as a filter.
It's not necessarily a problem to have it act this way - but it's not immeditately clear that it'll not act as a filter if it's not in "generator" mode.

Reply:
It was intentional, actually i thought about adding a option to chose if it is a filter or a addition to the current whitelist but that would be a bit confusing i think.